### ISSUE TITLE
Translate function 

### ISSUE THREAD

**Issue by Slabhub** [usoonees/logseq-plugin-whisper-subtitles#1]
Can be added translate subtitel future ?  with  google translate api  ?

**Comment by usoonees** [comment]
Yes, it could, maybe with google translate api or chatgpt api, would take a look later on.

**Comment by usoonees** [comment]
Sorry, I don't have the time to add this translation function. 

If someone could submit a PR, I'd be happy to merge it.
